src/components/KYCWall/BaseKYCWall.tsx

```diff
@@ -68,8 +67,8 @@ function KYCWall({
     walletTerms,
     shouldShowPersonalBankAccountOption = false,
 }: BaseKYCWallProps) {
-    const anchorRef = useRef<HTMLDivElement>(null);
-    const transferBalanceButtonRef = useRef<HTMLDivElement | null>(null);
+    const anchorRef = useRef<HTMLElement>(null);
```

Comment:
Why is this changed?

Reply:
It's related with this comment: https://github.com/Expensify/App/pull/33714#issuecomment-1899988170
In order to make the types compatible with MenuItem and Button we had to change the event type which than the target is not compatible with HTMLDivElement and just with HTMLElement.